src/modules/0.0.11/aws_iam/index.ts

```diff
@@ -46,13 +62,23 @@ export const AwsIamModule: Module2 = new Module2({
           for (const role of es) {
             const roleArn = await client.newRoleLin(
               role.roleName,
-              role.assumeRolePolicyDocument,
+              JSON.stringify(role.assumeRolePolicyDocument),
```

Comment:
Is this actually necessary, or was it a change you made while debugging that may not be necessary.

If it is necessary, how did we not catch this issue before, basically?

Reply:
oh no, this is because before the `role.assumeRolePolicyDocument ` was a `string` and now I've changed it to be `jsonb`. The AWS API still need it to be a `string`, though.

Reply:
Ahh, got it.